### ISSUE TITLE
mpd file created  with MP4BOX for live streaming not working

### ISSUE THREAD

**Issue by rofeifei** [gpac/gpac#421]
Hi,
I am trying to stream the video from an IP camera  using dash.js,the steps which i have tried is given below:

1.I am using avconv to capture from camera,i am using the follow command:
```
avconv -i "rtsp://[camera-ip-address]:554/mpeg4/media.amp"  -vcodec libx264 -vb 1800k -bufsize 3600k -keyint_min 192 -g 192 -vprofile main profile 
-acodec aac  -ar 48000 -ac 2 -strict experimental -f h264 live_h264_1800k.H264 \-vcodec libx264 -vb 2200k -bufsize 4400k -keyint_min 192 -g 192 -vprofile main -acodec aac  -ar 48000 -ac 2 -strict experimental -f h264 live_h264_2200k.h264

```
2.Then i am using mp4box to generated dynamic live .mpd and .m4s
i am using the follow command:
`MP4Box -dash-live 4000  -fps 24 -profile dashavc264:live  -dynamic  -mpd-refresh 1200 -bs-switching no -dash-ctx $ctxtOut  -segment-name dashseg%s   -out $pathMpdFile -add live_h264_1800k.h264  live_h264_2200k.h264   live_h264_1800k.mp4  live_h264_2200k.h264
`
3/i am using (1) and (2) in parallel
from dash-if player i am receiving this log(`No muxed data`.) and video does not start:

Please, can you give me some suggestion  to solve this issue?
thanks

**Comment by rbouqueau** [comment]
Duplicate of https://github.com/Dash-Industry-Forum/dash.js/issues/1008. Please don't waste open-source contributors' time. Thanks.

**Comment by rbouqueau** [comment]
You could also have a look at https://github.com/gpac/test-engine-live-tools

**Comment by rofeifei** [comment]
Hi,
@rbouqueau please you can see Dash-Industry-Forum/dash.js#1008 i have added some reasons&questions
thanks

**Comment by rbouqueau** [comment]
> You could also have a look at https://github.com/gpac/test-engine-live-tools

Have you tried the link above? It does what you are trying to do.

You cannot feed MP4Box with some live content. You need to feed MP4Box -live with pre-segmented chunks. There a feature-requests to support this.

An alternative that I don't recommend because it is not stable enough would be to use dashcast.
